pkg/gardenlet/controller/shoot/shoot/cleaner.go

```diff
@@ -184,6 +184,11 @@ func (c *cleaner) finalizeShootManagedResources(ctx context.Context, namespace s
 
 	shootMRList := &resourcesv1alpha1.ManagedResourceList{}
 	for _, mr := range mrList.Items {
+		// TODO(rfranzke): Uncomment the next line after v1.85 has been released
```

Comment:
```suggestion
		// TODO(rfranzke): Uncomment the next line after v1.90 has been released
```
Nit: Just in case someone else goes ahead and wants to prematurely clean up this snippet.